lib/text/ttml_text_parser.js

```diff
@@ -1061,7 +1061,7 @@ shaka.text.TtmlTextParser.RateInfo_ = class {
  * @example 50.17% 10%
  */
 shaka.text.TtmlTextParser.percentValues_ =
-    /^(\d{1,2}(?:\.\d+)?|100)% (\d{1,2}(?:\.\d+)?|100)%$/;
+    /^(\d{1,2}(?:\.\d+)?|100(?:.0+)?)% (\d{1,2}(?:\.\d+)?|100(?:.0+)?)%$/;
```

Comment:
I think the dots you added need to be escaped.  Otherwise, they mean "any character".

Reply:
@joeyparrish You are right. I have made changes.
Thanks!